Index: product-integrator/lib/vscode/build/lib/extensions.ts
===================================================================
--- product-integrator.orig/lib/vscode/build/lib/extensions.ts
+++ product-integrator/lib/vscode/build/lib/extensions.ts
@@ -270,11 +270,15 @@
 	return gulp.src(vsixPath)
 		.pipe(buffer())
 		.pipe(es.mapSync((f: File) => {
-			const hash = crypto.createHash('sha256');
-			hash.update(f.contents as Buffer);
-			const checksum = hash.digest('hex');
-			if (checksum !== sha256) {
-				throw new Error(`Checksum mismatch for ${vsixPath} (expected ${sha256}, actual ${checksum}))`);
+			if (typeof sha256 === 'string' && sha256.length > 0) {
+				const hash = crypto.createHash('sha256');
+				hash.update(f.contents as Buffer);
+				const checksum = hash.digest('hex');
+				if (checksum !== sha256) {
+					throw new Error(`Checksum mismatch for ${vsixPath} (expected ${sha256}, actual ${checksum}))`);
+				}
+			} else {
+				fancyLog(ansiColors.yellow(`[vsix-checksum] Missing expected checksum for ${vsixPath}; skipping checksum validation.`));
 			}
 			return f;
 		}))
