Index: product-integrator/lib/vscode/build/linux/dependencies-generator.ts
===================================================================
--- product-integrator.orig/lib/vscode/build/linux/dependencies-generator.ts
+++ product-integrator/lib/vscode/build/linux/dependencies-generator.ts
@@ -20,7 +20,7 @@ import product from '../../product.json'
 // If true, we fail the build if there are new dependencies found during that task.
 // The reference dependencies, which one has to update when the new dependencies
 // are valid, are in dep-lists.ts
-const FAIL_BUILD_FOR_NEW_DEPENDENCIES: boolean = true;
+const FAIL_BUILD_FOR_NEW_DEPENDENCIES: boolean = false;
 
 // Based on https://source.chromium.org/chromium/chromium/src/+/refs/tags/142.0.7444.175:chrome/installer/linux/BUILD.gn;l=64-80
 // and the Linux Archive build
